src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesService.kt

```diff
@@ -36,20 +36,24 @@ class ActivitiesService(
   private val objectMapper: ObjectMapper,
 ) : CreateMappingRetryable {
 
-  suspend fun createActivity(event: ScheduleDomainEvent) {
+  suspend fun createActivityEvent(event: ScheduleDomainEvent) {
+    createActivity(event.additionalInformation.activityScheduleId)
+  }
+
+  suspend fun createActivity(activityScheduleId: Long) {
```

Comment:
Now have 2 ways into the synchronisation, by event or called directly from the endpoint.